tests/test_input_outputs.py

```diff
@@ -22,6 +24,11 @@
 NUM_CLASSES = 2
 
 
+class MultiInputTransformer(FunctionTransformer):
+    def get_metadata(self):
+        return dict()
```

Comment:
Why do you need to define `MultipleInputTransformer`? It doesn't do anything.

Reply:
It is used [a couple of lines below](https://github.com/adriangb/scikeras/blob/678a40562a11ffe9900ca9d98579981b53150595/tests/test_input_outputs.py#L60):
```python3
@property
def feature_encoder(self):
    return MultiInputTransformer(func=lambda X: [X[:, 0], X[:, 1:4]],)
```

Reply:
Ah but yes, it **isn't** used because all it does is define `get_metadata` which we now made optional. Good catch.